manifests/init.pp

```diff
@@ -75,6 +79,33 @@
     path => $certs::iop_advisor_engine::client_ca_cert,
   }
 
+  podman::quadlet { $service_name:
```

Comment:
Personally, I'd rather just have an `epp` template that defines what we need rather than this "abstraction". But being consistent for the moment with how we tend to approach these.